doc/_admin-guide/200_About/README.md

```diff
@@ -12,13 +12,13 @@ description: >-
 ## Contacting us
 
 For sales and other inquiries, such as licensing, support, and renewals,
-visit <https://www.oneidentity.com/company/contact-us.aspx>.
+visit Contact Us page on our website.
 
 ## Technical support resources
 
 Technical support is available to One Identity customers with a valid
 maintenance contract and customers who have trial versions. You can
-access the Support Portal at <https://support.oneidentity.com/>.
+access the Support Portal [[here|oi-support-serv]].
```

Comment:
to eliminate here maybe we could use:


Technical support for One Identity customers with a valid maintenance contract, as well as for those using a trial version, is available via our [[Support Portal|oi-support-serv]].